octomap/octomap.pyx

```diff
@@ -810,7 +818,7 @@ cdef class OcTree:
 
     def dynamicEDT_update(self, updateRealDist):
         if self.edtptr:
-            return self.edtptr.update(<cppbool?>updateRealDist)
+            self.edtptr.update(<cppbool?>updateRealDist)
```

Comment:
def looks like:

```cython
cdef extern from "dynamicEDT3D/dynamicEDTOctomap.h":
    cdef cppclass DynamicEDTOctomap:
        ...
        void update(bool updateRealDist)
```

so i ran into error about cannot convert `void` to `PyObject` so i just rm return